### PR TITLE
enable e2e tests with turbopack

### PR DESCRIPTION
**Pull request by sokra**
Closes WEB-1862

### CODE REVIEW

**Review by kwonoj — DISMISSED** [review]

**Comment by ijjk** [comment]
## Failing test suites 
Commit: e5da6e82d089e146cdd6dcf009701b4e264f970d

`pnpm test-dev test/e2e/app-dir/app-config-crossorigin/index.test.ts` 
- app dir - crossOrigin config > should render correctly with assetPrefix: "/"

<details>
<summary>Expand output</summary>

  ● app dir - crossOrigin config › should render correctly with assetPrefix: "/"

    expect(received).toBe(expected) // Object.is equality

    Expected: "use-credentials"
    Received: undefined

      19 |       ).each((_, el) => {
      20 |         const crossOrigin = $(el).attr('crossorigin')
    > 21 |         expect(crossOrigin).toBe('use-credentials')
         |                             ^
      22 |       })
      23 |
      24 |       // Inline <script /> (including RSC payload) and <link /> should not have crossorigin attribute

      at Object.toBe (e2e/app-dir/app-config-crossorigin/index.test.ts:21:29)
      at initialize.Object.<anonymous>.exports.each (../node_modules/.pnpm/cheerio@0.22.0/node_modules/cheerio/lib/api/traversing.js:300:24)
      at Object.each (e2e/app-dir/app-config-crossorigin/index.test.ts:19:9)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

**Comment by ijjk** [comment]
# Stats from current PR

<details open>
<summary><strong>Default Build</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| buildDuration | 10.2s | 10.1s | N/A |
| buildDurationCached | 6s | 6s | N/A |
| nodeModulesSize | 175 MB | 175 MB | ✓ |
| nextStartRea..uration (ms) | 401ms | 398ms | N/A |

</details>

<details>
<summary><strong>Client Bundles (main, webpack)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| 199-HASH.js gzip | 32.3 kB | 32.4 kB | N/A |
| 3f784ff6-HASH.js gzip | 53.2 kB | 53.2 kB | N/A |
| 99.HASH.js gzip | 182 B | 182 B | ✓ |
| framework-HASH.js gzip | 45.5 kB | 45.5 kB | ✓ |
| main-app-HASH.js gzip | 253 B | 251 B | N/A |
| main-HASH.js gzip | 35.3 kB | 35.4 kB | N/A |
| webpack-HASH.js gzip | 1.75 kB | 1.75 kB | N/A |
| Overall change | 45.7 kB | 45.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| _app-HASH.js gzip | 206 B | 205 B | N/A |
| _error-HASH.js gzip | 182 B | 180 B | N/A |
| amp-HASH.js gzip | 506 B | 505 B | N/A |
| css-HASH.js gzip | 322 B | 323 B | N/A |
| dynamic-HASH.js gzip | 2.59 kB | 2.59 kB | ✓ |
| edge-ssr-HASH.js gzip | 260 B | 259 B | N/A |
| head-HASH.js gzip | 350 B | 350 B | ✓ |
| hooks-HASH.js gzip | 369 B | 369 B | ✓ |
| image-HASH.js gzip | 4.38 kB | 4.38 kB | N/A |
| index-HASH.js gzip | 256 B | 256 B | ✓ |
| link-HASH.js gzip | 2.67 kB | 2.67 kB | N/A |
| routerDirect..HASH.js gzip | 316 B | 318 B | N/A |
| script-HASH.js gzip | 385 B | 384 B | N/A |
| withRouter-HASH.js gzip | 319 B | 319 B | ✓ |
| 1afbb74e6ecf..834.css gzip | 106 B | 106 B | ✓ |
| Overall change | 3.99 kB | 3.99 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| _buildManifest.js gzip | 484 B | 482 B | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| index.html gzip | 528 B | 530 B | N/A |
| link.html gzip | 542 B | 543 B | N/A |
| withRouter.html gzip | 524 B | 524 B | ✓ |
| Overall change | 524 B | 524 B | ✓ |

</details>

<details>
<summary><strong>Edge SSR bundle Size</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| edge-ssr.js gzip | 96 kB | 96 kB | N/A |
| page.js gzip | 139 kB | 140 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Middleware size</strong></summary>

|  | vercel/next.js canary  | vercel/next.js sokra/e2e | Change |
| - | - | - | - |
| middleware-b..fest.js gzip | 620 B | 624 B | N/A |
| middleware-r..fest.js gzip | 150 B | 151 B | N/A |
| middleware.js gzip | 25.8 kB | 25.8 kB | N/A |
| edge-runtime..pack.js gzip | 1.92 kB | 1.92 kB | ✓ |
| Overall change | 1.92 kB | 1.92 kB | ✓ |

</details>

</details>
Commit: e5da6e82d089e146cdd6dcf009701b4e264f970d

File: test/turbopack-tests-manifest.json [hunk 1]
```diff
@@ -3308,21 +3310,21 @@
...
       "app dir - metadata dynamic routes social image routes should handle manifest.[ext] dynamic routes",
       "app dir - metadata dynamic routes social image routes should render og image with opengraph-image dynamic routes",
-      "app dir - metadata dynamic routes social image routes should render og image with twitter-image dynamic routes",
-      "app dir - metadata dynamic routes social image routes should support generate multi images with generateImageMetadata",
       "app dir - metadata dynamic routes text routes should handle robots.[ext] dynamic routes"
     ],
     "failed": [
       "app dir - metadata dynamic routes should error when id is missing in generateSitemaps",
       "app dir - metadata dynamic routes should generate unique path for image routes under group routes",
+      "app dir - metadata dynamic routes should inject dynamic metadata properly to head",
```
Comment (kwonoj) [inline test/turbopack-tests-manifest.json:3320]: https://github.com/vercel/next.js/pull/57431 is in canary, so rebasing should make pass some of these tests

Comment (sokra) [reply test/turbopack-tests-manifest.json:3320]: It's updated from the daily test run... The next run will make it pass.

Comment (kwonoj) [reply test/turbopack-tests-manifest.json:3320]: yeah but there's passing test and don't need to move to failed manually? but it's indifferent

**Review by kwonoj — APPROVED** [review]

**Pull request merged**